$NetBSD$

Add directory entry information. Without this, ${PREFIX}/info/dir
won't be updated by install-info. Typical error message will be:
  ----
% install-info --dir-file=/usr/pkg/info/dir ./work/simulavr-1.0.0/doc/simulavr.info
install-info: warning: no info dir entry in `./work/simulavr-1.0.0/doc/simulavr.info'
  ----

--- doc/simulavr.texinfo.orig	2012-02-13 00:26:38.000000000 +0900
+++ doc/simulavr.texinfo	2013-08-14 14:04:34.000000000 +0900
@@ -36,6 +36,11 @@ by the Free Software Foundation.
 
 @end copying
 
+@dircategory SimulAVR
+@direntry
+* Simulavr: (SimulAVR). A simulator for the Atmel AVR family of microcontrollers.
+@end direntry
+
 @titlepage
 @title SimulAVR - an AVR simulation framework
 @subtitle A simulator for the Atmel AVR family of microcontrollers.
